README.md

```diff
@@ -132,7 +132,7 @@ of which can be seen [here](https://github.com/wikispeedruns/wikipedia-speedruns
 Unfortunately, neither celery nor redis are supported on windows. So if you have
 a windows development machine, you will have to run the server through WSL. Note
 that if you want to keep your windows MySQL instance, you need to figure out
-which port the host windows machine is exposed on in WSL. See [this Stack Overflow
+which port the host windows machine is exposed on in WSL. See [this Super User
```

Comment:
Hmm, Stack Overflow is a lot more well-known/clear than Super User, Super User is also a part of Stack Exchange. Maybe we can change it to just say "See this post"